src/07-even-more-loops.js

```diff
@@ -143,7 +252,18 @@ function findEqual(array, selected) {}
  *  //> [ 1, 11, 2, 3, 4, 9 ]
  */
 
-function removeDuplicates(array) {}
+
+function removeDuplicates(array) {
+  let newArr = []
+  for (let i = 0; i <array.length; i++){
+    if (newArr.includes(array[i])=== false) {
+      newArr.push(array[i])
+    }
+  }
+  return newArr 
+
+}
+
```

Comment:
Great solution! We'll explore a more efficient solution when we talk algorithms down the road.